docs/vpc-configuration.md

```diff
@@ -0,0 +1,159 @@
+# VPC Configuration
+
+### Contents
+
+##### [1. Basic configuration](#basic-configuration)
+
+VPC configuration is done in the under the 'vpc' key section of the cloudformation configuration file.
+
+Defaults are set so that without specifying anything, we get a VPC with CIDR `10.128.0.0/16` and 3 subnets, `10.0.0.0/20`, `10.0.16.0/20`, and `10.0.32.0/20`
+
+We can alter this by specifying the CIDR and subnets specically, for example,
+
+	vpc:
+	    CIDR: 10.128.0.0/16
+	    SubnetX: 10.128.0.0/20
+	    SubnetY: 10.128.16.0/20
+
+##### [2. Peering](#peering)
+
+##### [A1. Examples](#examples)
+
+* [Peered VPCs](#peered-vpcs)
+
+### Basic configuration 
+
+
+### Peering
+
+Using the VPC class, we can setup up peering to another VPC by calling enable_peering. This call will then
+
+* Use the cloudformation configuration file to get a list of stacks to peer to
```

Comment:
Does this mean that one can only peer to stacks defined in the same .yml file ? (and not ones from other repos)